labrad/node/__init__.py

```diff
@@ -548,14 +552,17 @@ def _load(self, nodename=None, create=False, defaults=None):
             p.set('directories', defaults[0])
             p.set('packages', defaults[1])
             p.set('extensions', defaults[2])
+            p.set('autostart', defaults[3])
         p.get('directories', '*s', key='dirs')
         p.get('packages', '*s', key='mods')
         p.get('extensions', '*s', key='exts')
+        p.get('autostart', '*s', True, [], key='autostart')
         ans = yield p.send()
         dirs = filter(None, ans.dirs)
         mods = filter(None, ans.mods)
         exts = filter(None, ans.exts)
-        returnValue((dirs, mods, exts))
+        autostart = sorted(filter(None, ans.autostart))
```

Comment:
That's a little obscure. I had to look up `filter` and then the special case where the first argument is `None`. List comprehension might be easier to understand.